GiniVision/Classes/Core/Custom views/NoticeView.swift

```diff
@@ -14,11 +14,9 @@ import UIKit
     var title: String {
         switch self {
         case .retry:
-            return NSLocalizedStringPreferred("ginivision.multipagereview.error.retryAction",
-                                               comment: "button title for retry action")
+            return .localized(resource: MultipageReviewStrings.retryActionButton)
```

Comment:
You moved the comment to the *Strings enum. I feel that you lost some additional hint here and in the enum, it's not really helpful. It would be better to see where hints at the place these strings are used (hence, here).

I wouldn't say that these comments are really critical, but since we already have them, I'd rather keep them at the place where they are used.

Reply:
I thought about it and I found better to have them separately, mostly for those times when there is more than one localized strings in a row.  In makes the code more readable and clear (most of the times you have to split the comment into 2-3 lines because it does not fit the line width)